### PR TITLE
adjust domain of subscriptions.eventing CRD to kyma-project.io resources

### LINKED ISSUE #407: Adjust domain of subscriptions.eventing CRD to 'kyma-project.io'
The CRD 'subscriptions.eventing.kyma.cx' is using kyma.cx as domain (group). As kyma.cx was intended to be used for the commercialization aspects of kyma.
 The official domain of kyma is kyma-project.io now, so all CRDs should get adjusted to it.

Prerequisite:
 * -Align with automatix before starting this task.-
 * See YSF-3178

Note:
 * The group will be changed from 
   * {{eventing.kyma.cx ---> eventing.kyma-project.io}}
 * The name will be changed from
   * {{subscriptions.eventing.kyma.cx ---> subscriptions.eventing.kyma-project.io}}

Workflow
 * Do the code changes and regenerate the code
 * Push the changes to the master
 * Get the new image generated
 * Helm chart changes
   * Change the CRD and use the new image.
   * Align with automatix to get their changes.
 * Push to master.

 

AC:
 - group name is adjusted in the CRD
 - all usage is adjusted
 - documentation and examples are adjusted
 - The [e2e flow](https://github.com/kyma-incubator/examples/tree/master/kyma-local-e2e-scenario) with lambda and gateway integration is working after the changes.

### PR DESCRIPTION
**Pull request by montaro**
**Related issue(s)**
#407 
#1624

### CODE REVIEW

File: resources/core/values.yaml [hunk 1]
```diff
@@ -53,11 +53,11 @@ global:
     dir: develop/
     version: 6e0a7fe2
   event_bus:
-    dir: develop/
-    version: f9df7d26
+    dir: pr/
+    version: PR-1624
```
Comment (marcobebway) [inline resources/core/values.yaml]: I believe this is the wrong PR number.

File: resources/core/values.yaml [hunk 2]
```diff
@@ -53,11 +53,11 @@ global:
...
   event_bus_tests:
-    dir: develop/
-    version: 1daa11b7
+    dir: pr/
+    version: PR-1624
```
Comment (marcobebway) [inline resources/core/values.yaml]: Also here, wrong PR number.

Comment (montaro) [reply resources/core/values.yaml]: I believe this file shouldn't be part of this PR and also the images shouldn't be merged to master where it's configured to be pulled from a PR directory.
I will remove this file of this PR, and after the merge, I will create a new PR for the tests. then a new PR for the resources.

**Comment by montaro** [comment]
@marcobebway I'm reproducing and here are the failure logs:
```
➜  ~ kubectl logs test-core-kubeless --namespace kyma-system
2018/11/24 00:43:56 Cleaning up
2018/11/24 00:43:57 Current state of the ns:kubeless-test is:
 warning: Immediate deletion does not wait for confirmation that the running resource has been terminated. The resource may continue to run on the cluster indefinitely.
2018/11/24 00:43:57 Unable to delete:
warning: Immediate deletion does not wait for confirmation that the running resource has been terminated. The resource may continue to run on the cluster indefinitely.
unable to recognize "k8syaml/k8s.yaml": no matches for kind "Subscription" in version "eventing.kyma.cx/v1alpha1"
```
`unable to recognize "k8syaml/k8s.yaml": no matches for kind "Subscription" in version "eventing.kyma.cx/v1alpha1"`

**Comment by marcobebway** [comment]
@montaro kubeless tests will be fixed as part of #1769. 
Event-Bus tests should be fixed in the current ticket.

File: docs/event-bus/docs/030-cr-subscription.md [hunk 3]
```diff
@@ -3,20 +3,20 @@ title: Subscription
 type: Custom Resource
 ---
 
-The `subscriptions.eventing.kyma.cx` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to create an event trigger for lambda or microservice in Kyma. After creating a new custom resource, the event trigger is registered in the event bus and events are delivered to the endpoint specified in the custom resource.
+The `subscriptions.eventing.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to create an event trigger for lambda or microservice in Kyma. After creating a new custom resource, the event trigger is registered in the event bus and events are delivered to the endpoint specified in the custom resource.
```
Comment (bszwarc) [inline docs/event-bus/docs/030-cr-subscription.md]:
```suggestion
The `subscriptions.eventing.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to create an Event trigger for a lambda or microservice in Kyma. After creating a new custom resource, the Event trigger is registered in the Event Bus and Events are delivered to the endpoint specified in the custom resource.
```

**Comment by sayanh** [comment]
LGTM, as testing in the following areas were successful:
- lambda-ui
- examples(event-subscription/lambda)
- kubeless-int tests

**Pull request merged**